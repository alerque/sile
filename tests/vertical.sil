\begin[class=jplain,layout=tate]{document}
\font[family=Noto Sans CJK JP,language=ja]%
日本は昔から
\end{document}
