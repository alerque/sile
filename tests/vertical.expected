Set paper size 	595.275597	841.8897729
Begin page
Mx 	528.4079
My 	107.6592
Set font 	Noto Sans CJK JP;24;400;;normal;;TTB
T	20220	(日)
My 	131.9203
T	20758	(本)
End page
Finish
